tests/agents/arbitrage_agent/test_arbitrage_agent.py

```diff
@@ -0,0 +1,63 @@
+import typing as t
+from unittest.mock import Mock, patch
+
+import pytest
+from prediction_market_agent_tooling.markets.agent_market import AgentMarket
+from prediction_market_agent_tooling.markets.omen.omen import OmenAgentMarket
+
+from prediction_market_agent.agents.arbitrage_agent.deploy import (
+    DeployableArbitrageAgent,
+)
+
+
+@pytest.fixture(scope="module")
+def arbitrage_agent() -> t.Generator[DeployableArbitrageAgent, None, None]:
+    with patch(
+        "prediction_market_agent.agents.arbitrage_agent.deploy.DeployableArbitrageAgent.load",
+        new=lambda x: None,
+    ), patch(
+        "prediction_market_agent_tooling.tools.langfuse_.get_langfuse_langchain_config"
+    ):
+        agent = DeployableArbitrageAgent()
+        # needed since load was mocked
```

Comment:
I didn't see you did anything in the `load` method?

But I agree that the stuff that's done in overloaded `run` method were meant to be done in the `load` method.

Reply:
I discussed about this somewhere with @evangriffiths - this agent is Omen-specific, and the "is this Omen else Raise" is done on the run method (whereas `load` is called during Init, which in turn is called in `run_agent.py` before market_type is passed as arg). Hence the `self.attributes` being defined here.

Reply:
Sure, sounds as a good reason to me to not use `load` method, but then this comment is incorrect.